src/main/cpp/t01_max.cpp

```diff
@@ -27,6 +27,24 @@
 using namespace std;
 
 int t01_max() {
-    
-    return 0;
+    int n,m;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'm' is not assigned a value.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4729584)